open-api/swagger.json

```diff
@@ -2263,6 +2263,12 @@
                 },
                 "bookmarked": {
                   "type": "boolean"
+                },
+                "subscribed": {
```

Comment:
I can also change the UserQueryController to prevent having a situation like this. So it will ensure that: if subscribed = false, subscriptionFreq has to be null.
Would it solve the issue?

Reply:
I am not sure whether it's even the issue. We could leave it like that.